rllib/agents/alpha_star/alpha_star.py

```diff
@@ -297,13 +313,15 @@ def training_iteration(self) -> ResultDict:
         # - Rollout results are sent directly to correct replay buffer
         #   shards, instead of here (to the driver).
         with self._timers[SAMPLE_TIMER]:
-            sample_results = asynchronous_parallel_requests(
-                remote_requests_in_flight=self.remote_requests_in_flight,
-                actors=self.workers.remote_workers() or [self.workers.local_worker()],
-                ray_wait_timeout_s=self.config["sample_wait_timeout"],
-                max_remote_requests_in_flight_per_actor=2,
-                remote_fn=self._sample_and_send_to_buffer,
-            )
+            # if there are no remote workers (e.g. num_workers=0)
+            if not self.workers.remote_workers():
+                worker = self.workers.local_worker()
+                statistics = worker.apply(self._sample_and_send_to_buffer())
```

Comment:
do you really mean to call the function here?

Reply:
yeah I don't my bad